package.json

```diff
@@ -38,5 +38,9 @@
     "@jupiterone/integration-sdk-dev-tools": "^8.4.4",
     "@types/js-yaml": "^4.0.5",
     "@types/nodegit": "^0.27.7"
+  },
+  "volta": {
+    "node": "14.19.0",
```

Comment:
I've never used Volta before, but according to the docs, it seems like Volta supports major versions to be supplied? What do you think about not requiring 14.19.0 _precisely_, but instead just any minor or patch version under major 14?

See: https://docs.volta.sh/guide/understanding#managing-your-project

Reply:
Same question may apply to `yarn` below

Reply:
I presumed the same... but apparently Volta adheres to an explicit version number.  Here is a thread that discusses the design decision: https://github.com/volta-cli/volta/issues/742

Im fine to close the PR without merging.  Just testing the waters in contributing. :P

Reply:
I'm good with it! Thank you for the info!